docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: "3"
 services:
   kutt:
     image: kutt/kutt
+    #build: .
```

Comment:
does this need to be commented out?

Reply:
No,  can be omitted. It's just for building local containers from local code